src/main/java/teammates/common/util/ActivityLogEntry.java

```diff
@@ -386,10 +386,7 @@ public String getActionInfo(){
         return "<a href=\""+getUrlToShow()+"\" class=\""+style+"\" target=\"_blank\">"+servletName+"</a>";
     }
     
-    public String getMessageInfo(){
-        
-        Sanitizer.sanitizeForHtml(message);
-        
+    public String getMessageInfo(){        
```

Comment:
Since the change has affected this line, make sure to leave the code ```better than you saw it```, space after ```)``` and before ```{```